src/main/java/org/terasology/commonworld/geom/BoundingBox.java

```diff
@@ -16,17 +16,18 @@
 
 package org.terasology.commonworld.geom;
 
+import com.google.common.base.Optional;
```

Comment:
Could we use Java's `Optional` here? I think this code predates Java 8, thus it wasn't available back then...